readme.md

```diff
@@ -16,35 +16,49 @@ $ npm install --save hook-std
 const assert = require('assert');
 const hookStd = require('hook-std');
 
-const unhook = hookStd.stdout(output => {
+const promise = hookStd.stdout(output => {
+	promise.unhook();
+
+	assert.strictEqual(output.trim(), 'unicorn');
+});
+
+console.log('unicorn');
+await promise;
+```
+
+Unhook can also be done via callback:
+
+```js
+const promise = hookStd.stdout((output, encoding, unhook) => {
```

Comment:
Drop the `encoding` parameter (from here and the code). I don't see the value of it in this module. This module is meant to operate on strings. Instead, just convert it to a string, if it's not already, before passing it into the `transform` function. If the `transform` function returns a string, convert it to the existing encoding, otherwise just pass it through with the correct encoding.